apps/next-evans/components/pages/explore/AudioBooksBlock.tsx

```diff
@@ -11,19 +11,7 @@ import { getDocumentUrl, isCompilations } from '@/lib/friend';
 import { mostModernEditionType } from '@/lib/editions';
 
 interface Props {
-  books: Array<
-    Pick<
-      DocumentWithMeta,
-      | 'title'
-      | 'slug'
-      | 'editions'
-      | 'customCSS'
-      | 'customHTML'
-      | 'isbn'
-      | 'authorSlug'
-      | 'authorName'
-    >
-  >;
+  books: Array<Doc<'editions'>>;
```

Comment:
this generic utility type `Doc<T>` i think keeps the upside of always using `Pick`, while allowing for much simpler to write types.